packages/onchainkit/src/fund/components/FundButton.tsx

```diff
@@ -168,33 +169,23 @@ export function FundButton({
             {buttonIcon}
           </span>
         )}
-        {hideText || (
-          <span data-testid="ockFundButtonTextContent">
-            {buttonTextContent}
-          </span>
-        )}
+        <span data-testid="ockFundButtonTextContent">{buttonTextContent}</span>
       </>
     );
-  }, [buttonState, buttonIcon, buttonTextContent, hideText]);
-
-  if (openIn === 'tab') {
-    return (
-      <a
-        className={classNames}
-        href={fundingUrlToRender}
-        // If openIn is 'tab', default target to _blank so we don't accidentally navigate in the current tab
-        target={target ?? '_blank'}
-        rel={rel}
-      >
-        {buttonContent}
-      </a>
-    );
-  }
+  }, [buttonState, buttonIcon, buttonTextContent]);
 
   if (shouldShowConnectWallet) {
     return <ConnectWallet className={cn('w-full', className)} />;
   }
 
+  if (render) {
+    return render({
+      state: buttonState,
```

Comment:
follow-up: I think this is named `status` in the Transaction PR. Assuming it's more or less doing the same thing, we should standardize that across components

Reply:
I think I prefer status, but no strong feelings